extras/tests/MsgPackSerializer/serializeVariant.cpp

```diff
@@ -146,6 +147,19 @@ TEST_CASE("serialize MsgPack value") {
     checkVariant(serialized("\xDB\x00\x01\x00\x00", 5), "\xDB\x00\x01\x00\x00");
   }
 
+  SECTION("bin 8") {
+    static const auto bin8 = std::array<char, 4>({1, 2, 3, 4});
+    checkVariant(MsgPackBinary(bin8.data(), bin8.size()),
+                 "\xC4\x04\x01\x02\x03\x04");
+  }
+
+  SECTION("bin 16") {
+    static const auto bin16 = std::array<char, 0x100>({1, 2, 3, 4});
+    std::string bin16string(bin16.data(), bin16.size());
```

Comment:
You could simply use `std::string(0x100, '?')`; this constructor creates a string that repeats the provided character N times.
See https://en.cppreference.com/w/cpp/string/basic_string/basic_string (2)